spdx-utils/src/main/kotlin/SpdxLicenseStringMapping.kt

```diff
@@ -289,13 +293,15 @@ object SpdxLicenseStringMapping {
             "Zope Public" to ZPL_2_1.toExpression(),
             "http://ant-contrib.sourceforge.net/tasks/LICENSE.txt" to APACHE_1_1.toExpression(),
             "http://opensource.org/licenses/MIT" to MIT.toExpression(),
+            "http://polymer.github.io/LICENSE.txt" to BSD_3_CLAUSE.toExpression(),
             "http://svnkit.com/license.html" to TMATE.toExpression(),
             "http://www.apache.org/licenses/LICENSE-2.0" to APACHE_2_0.toExpression(),
             "http://www.apache.org/licenses/LICENSE-2.0.txt" to APACHE_2_0.toExpression(),
             "http://www.gnu.org/copyleft/lesser.html" to LGPL_3_0_ONLY.toExpression(),
+            "https://raw.github.com/RDFLib/rdflib/master/LICENSE" to BSD_3_CLAUSE.toExpression(),
             "new BSD" to BSD_3_CLAUSE.toExpression(),
             "public domain, Python, 2-Clause BSD, GPL 3 (see COPYING.txt)"
-                    to (((licenseRef("scancode-public-domain-disclaimer") and PYTHON_2_0.toExpression())
+                    to (((licenseRef("public-domain") and PYTHON_2_0.toExpression())
```

Comment:
This should be squashed to 91d00de2ef5272ffc5564a840b6d1abdefbbc5ca IMO.

Reply:
Reverted, as discussed offline we use "scancode-public-domain-disclaimer" now.